pkg/hostagent/event_poster.go

```diff
@@ -173,8 +160,24 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	if !srcOk && !dstOk {
 		return nil
 	}
-
-	// update poster's eventSubmitTimeMap
-	agent.poster.eventSubmitTimeMap[packetEvent.SourceIP+packetEvent.DestinationIP] = currTime
+	if eventDestination != nil {
+		var eventSrcName, eventDstName string = packetEvent.SourceIP, packetEvent.DestinationIP
```

Comment:
Nit: Above lines 160-162, you are checking for srcOk and dskOk  if they are not present you are returning from there,    then you can  directly set the eventSrc and dst names right ?

Reply:
The pods may be present(srcOk,dstOk), but not in running state, we don't post events in that case.